### PR TITLE
[Merged by Bors] - doc: fix typos

### PR DESCRIPTION
**Pull request by pitmonticone**
Fix minor typos in the following files: 

- [x] `Mathlib/GroupTheory/GroupAction/Opposite.lean`
- [x] `Mathlib/Init/Control/Lawful.lean`
- [x] `Mathlib/ModelTheory/ElementarySubstructures.lean`
- [x] `Mathlib/Algebra/Group/Defs.lean` 
- [x] `Mathlib/Algebra/Group/WithOne/Basic.lean`
- [x] `Mathlib/Data/Int/Cast/Defs.lean`
- [x] `Mathlib/LinearAlgebra/Dimension/Basic.lean`
- [x] `Mathlib/NumberTheory/NumberField/CanonicalEmbedding.lean`
- [x] `Mathlib/Algebra/Star/StarAlgHom.lean`
- [x] `Mathlib/AlgebraicTopology/SimplexCategory.lean`
- [x] `Mathlib/CategoryTheory/Abelian/Homology.lean`
- [x] `Mathlib/CategoryTheory/Sites/Grothendieck.lean`
- [x] `Mathlib/RingTheory/IsTensorProduct.lean`
- [x] `Mathlib/AlgebraicTopology/DoldKan/Homotopies.lean`
- [x] `Mathlib/AlgebraicTopology/ExtraDegeneracy.lean`
- [x] `Mathlib/AlgebraicTopology/Nerve.lean`
- [x] `Mathlib/AlgebraicTopology/SplitSimplicialObject.lean`
- [x] `Mathlib/Analysis/ConstantSpeed.lean`
- [x] `Mathlib/Analysis/Convolution.lean`

### CODE REVIEW

File: Mathlib/Analysis/ConstantSpeed.lean [hunk 1]
```diff
@@ -12,7 +12,7 @@ import Mathlib.Analysis.BoundedVariation
 # Constant speed
 
 This file defines the notion of constant (and unit) speed for a function `f : ℝ → E` with
-pseudo-emetric structure on `E` with respect to a set `s : Set ℝ` and "speed" `l : ℝ≥0`, and shows
+pseudo-metric structure on `E` with respect to a set `s : Set ℝ` and "speed" `l : ℝ≥0`, and shows
```
Comment (mcdoll) [inline Mathlib/Analysis/ConstantSpeed.lean:15]: This is correct in the original version.

Comment (pitmonticone) [reply Mathlib/Analysis/ConstantSpeed.lean:15]: Ops, sorry. I'm going to correct it.

Comment (pitmonticone) [reply Mathlib/Analysis/ConstantSpeed.lean]: @mcdoll Done.

**Comment by mcdoll** [comment]
Thanks!
maintainer merge

**Pull request closed without merging**